### ISSUE TITLE
[Suggestion] King of the Hill Goal

### ISSUE THREAD

**Issue by mibby** [slipcor/pvparena#107]
Similar to domination but instead of having all 'flags' active at once, rotate between zone radiuses every defined (configurable) amount of time.

When the zone is being contested, revert to white.
When the zone has only one team in it, start claiming points for that team. (Defined, example every tick = 1 point) First team to x amount of points wins, count upward not down.

As well as have a visual representation of the radius capture size in some way, shape, or form? For example, a circular hologram border or something based on the team color or white for unclaimed/contested?

**Comment by slipcor** [comment]
v1.3.2.99 does all the things :D

I added some mechanics that you might like!

First off, the only difference for the moment is that it is "one active flag" which is indicated by a beacon. This beacon will either show when claimed (default) and score like domination (every 3 seconds, configurable) or have the server determine another beacon (and score a point).

Furthermore, every 30 seconds (configurable, disable-able) it changes the active beacon :)

I did my best to test it with my second account :) but I bet your players find a way to break it :P
